CONTRIBUTING.md

```diff
@@ -58,3 +59,19 @@ document, granting the dNation Kubernetes Monitoring project to use your work.
   limitations under the License.
 */
 ```
+
+## Release policy
+
+* Release branch `vMajor.Minor.x` (e.g. `v1.1.x`) - all PRs with new functionality should target  this branch
```

Comment:
- Double space `target  this`
- Use `-` instead of `*`
- Remove empty lines between paragraphs

Reply:
done